irohad/ametsuchi/impl/postgres_wsv_command.cpp

```diff
@@ -314,8 +314,12 @@ namespace iroha {
     WsvCommandResult PostgresWsvCommand::insertPeer(
         const shared_model::interface::Peer &peer) {
       soci::statement st = sql_.prepare
-          << "INSERT INTO peer(public_key, address, tls_certificate)"
-             " VALUES (lower(:pk), :address, :tls_certificate)";
+          << (peer.isSyncingPeer()
+                  ? "INSERT INTO sync_peer(public_key, address, "
+                    "tls_certificate) VALUES (lower(:pk), :address, "
+                    ":tls_certificate)"
+                  : "INSERT INTO peer(public_key, address, tls_certificate) "
+                    "VALUES (lower(:pk), :address, :tls_certificate)");
```

Comment:
The only difference in both SQL queries: is `sync_peer` vs `peer`, so maybe the condition `peer.isSyncingPeer()` can impact only that word in the SQL statement? (I didn't compiled its):
```suggestion
          << (format("INSERT INTO {}(public_key, address, "
                    "tls_certificate) VALUES (lower(:pk), :address, "
                    ":tls_certificate)", peer.isSyncingPeer() ? "sync_peer", "peer");
```